cardano-cli/src/Cardano/CLI/Shelley/Run/Transaction.hs

```diff
@@ -45,6 +45,7 @@ import qualified Cardano.Binary as CBOR
 import           Cardano.Ledger.Shelley.Scripts ()
 import           Cardano.Ledger.ShelleyMA.TxBody ()
 
+import           Cardano.Api.EraCast
```

Comment:
This should not be exposed to `cardano-cli`. Whatever you need should be exposed via `Cardano.Api` or `Cardano.Api.Shelley`.